pkg/test/expectations/expectations.go

```diff
@@ -99,6 +99,11 @@ func ExpectNodeExists(ctx context.Context, c client.Client, name string) *v1.Nod
 	return ExpectExists(ctx, c, &v1.Node{ObjectMeta: metav1.ObjectMeta{Name: name}})
 }
 
+func ExpectNodeClaimExists(ctx context.Context, c client.Client, name string) *v1beta1.NodeClaim {
```

Comment:
I don't see this used in this PR

Reply:
+1, can't you just use `ExpectExists` directly?

Reply:
Good call, missed this as part of a past revision